doc/source/whatsnew/v1.2.0.rst

```diff
@@ -488,23 +487,23 @@ Deprecations
 Performance improvements
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-- Performance improvements when creating DataFrame or Series with dtype ``str`` or :class:`StringDtype` from array with many string elements (:issue:`36304`, :issue:`36317`, :issue:`36325`, :issue:`36432`, :issue:`37371`)
-- Performance improvement in :meth:`GroupBy.agg` with the ``numba`` engine (:issue:`35759`)
-- Performance improvements when creating :meth:`pd.Series.map` from a huge dictionary (:issue:`34717`)
+- Performance improvements when creating :class:`DataFrame` or :class:`Series` with dtype ``str`` or :class:`StringDtype` from array with many string elements (:issue:`36304`, :issue:`36317`, :issue:`36325`, :issue:`36432`, :issue:`37371`)
```

Comment:
Also for example in this sentence, I would leave DataFrame/Series as they were (IMO it makes the sentence more difficult to read if there is a lot of different formatting).